src/components/pdf-export/pdf-document.tsx

```diff
@@ -0,0 +1,539 @@
+import React from "react";
+import { Page, Text, View, Document, StyleSheet, Font } from "@react-pdf/renderer";
+import { SurveySummary } from "../../types/index";
+import strings from "localization/strings";
+import theme from "theme";
+import OswaldBold from "../../resources/fonts/oswald/Oswald-Bold.ttf";
+import QuicksandRegular from "../../resources/fonts/quicksand/Quicksand-Regular.ttf";
+import QuicksandBold from "../../resources/fonts/quicksand/Quicksand-Bold.ttf";
+import LocalizationUtils from "utils/localization-utils";
+import { Survey } from "../../generated/client/models/Survey";
+import moment from "moment";
+
+// Register Oswald font
+Font.register({
+  family: "Oswald",
+  format: "truetype",
+  fonts: [
+    {
+      src: OswaldBold,
+      fontWeight: "bold"
+    }
+  ]
+});
+
+// Register Quicksand font
+Font.register({
+  family: "Quicksand",
+  format: "truetype",
+  fonts: [
+    {
+      src: QuicksandRegular,
+      fontWeight: "normal"
+    },
+    {
+      src: QuicksandBold,
+      fontWeight: "bold"
+    }
+  ]
+});
+
+const styles = StyleSheet.create({
+
+  page: {
+    paddingTop: theme.spacing(4),
+    paddingBottom: theme.spacing(8),
+    paddingHorizontal: theme.spacing(6),
+    fontSize: 12,
+    fontFamily: "Quicksand"
+  },
+
+  header: {
+    display: "flex",
+    textAlign: "center",
+    justifyContent: "space-between",
+    flexDirection: "row",
+    paddingBottom: theme.spacing(2)
+  },
+
+  documentHeaderText: {
+    fontFamily: "Oswald",
+    fontSize: 14,
+    color: "rgba(0,0,0,0.6)"
+  },
+
+  titleText: {
+    fontSize: 18,
+    fontFamily: "Oswald",
+    marginBottom: theme.spacing(2)
+  },
+
+  subtitle: {
+    fontSize: 16,
+    fontFamily: "Oswald",
+    marginBottom: theme.spacing(2)
+  },
+
+  bold: {
+    fontFamily: "Quicksand",
+    fontSize: 14,
+    fontWeight: "bold",
+    marginBottom: theme.spacing(0.5)
+  },
+
+  materialItem: {
+    display: "flex",
+    borderBottom: "1px solid #ddd",
+    padding: theme.spacing(2)
+  },
+
+  container: {
+    padding: theme.spacing(2)
+  },
+
+  row: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    marginBottom: theme.spacing(1),
+    paddingBottom: theme.spacing(0.5),
+    borderBottom: "1px dashed #ddd"
+  },
+
+  cell: {
+    borderBottom: "1px solid #ddd",
+    paddingBottom: theme.spacing(1),
+    marginBottom: theme.spacing(1)
+  }
+
+});
+
+interface Props {
+  selectedSurvey: Survey | undefined;
```

Comment:
Why is this optional? We cannot be in this screen if we don't have existing survey. Make this mandatory and remove all unnecessary undefined checks for survey